packages/blocks/flexera/src/auth.ts

```diff
@@ -4,10 +4,10 @@ const markdown = `
 Authenticate with your Flexera API Token to access Flexera services.
 `;
 
-export const flexeraAuth = BlockAuth.SecretAuth({
+export const flexeraSpotAuth = BlockAuth.SecretAuth({
   authProviderKey: 'Flexera',
-  authProviderDisplayName: 'Flexera',
-  authProviderLogoUrl: `https://static.openops.com/blocks/flexera.png`,
+  authProviderDisplayName: 'Flexera Spot',
```

Comment:
changed just the display name and logo, the key is the same for any existing connections